chat-plugins/uno.js

```diff
@@ -0,0 +1,563 @@
+"use strict";
+
+const maxTime = 20; // seconds
+
+function cardHTML(card, fullsize) {
+	let surface = card.value.replace(/[^A-Z0-9\+]/g, "");
+	return `<button style="height: 140px; padding-bottom: 120px; text-align: left; font-weight: bold; border: 2px solid rgba(33 , 68 , 72 , 0.59); min-width: ${fullsize ? "80" : "30"}px; border-radius: 10px 2px 2px 3px; color: ${card.colour === "Black" ? "yellow" : "black"}; background-color: ${card.colour}" name=send value="/uno play ${card.name}">${surface}</button>`;
```

Comment:
I think it was that we should probably try to avoid it if possible, but again, not sure